client/src/main/java/org/asynchttpclient/ClientStats.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2010 Ning, Inc.
+ *
+ * This program is licensed to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.asynchttpclient;
+
+import java.util.Objects;
+
+/**
+ * A record class representing the state of an (@link org.asynchttpclient.AsyncHttpClient)
+ */
+public class ClientStats {
+
+    private final long activeConnectionCount;
+    private final long idleConnectionCount;
+
+    public ClientStats(final long activeConnectionCount,
+                       final long idleConnectionCount) {
```

Comment:
Just a small nit. But considering the code below, the finals are pretty useless here. God, Scala gets it right here (immutability is the default)...